platform/network/iot_network_mbedtls.c

```diff
@@ -32,6 +32,15 @@
 #include <stdio.h>
 #include <string.h>
 
+#if defined( __GNUC__ )
+#include <netdb.h>
+#include <arpa/inet.h>
+#else
+/* Windows sockets */
+#include <winsock2.h>
```

Comment:
These don't need to be included by this header.  They're already handled by the porting files
```suggestion
```

Reply:
This is not the header but C file, I have not added any WSL headers in other headers.